host-interaction/file-system/write/write-file.yml

```diff
@@ -11,22 +13,37 @@ rule:
       # ntdll
       - 563653399B82CD443F120ECEFF836EA3678D4CF11D9B351BB737573C2D856299:0x1400025C4
   features:
```

Comment:
this file is getting a bit unruly. the logic tree is so deep that its tricky to interpret as a human.

we should probably split this file into "write file on POSIX" and "write file on Windows".